language/tools/move-mv-llvm-compiler/src/main.rs

```diff
@@ -35,6 +35,8 @@ use std::path::PathBuf;
 
 use move_mv_llvm_compiler::package::resolve_dependency;
 
+use move_command_line_common::address::ParsedAddress;
```

Comment:
It would be better to preserve the lexicographic ordering of uses. Probably the linter could enforce this.

Reply:
Beautifications outside of the linter should be in a separate PR.

Reply:
I removed long standing `#![allow(unused)]` and made some clean up.